.github/workflows/vapoursynth.yml

```diff
@@ -0,0 +1,151 @@
+name: vapoursynth
+
+on: [push, pull_request]
+
+jobs:
+  clippy-rustfmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+        components: clippy, rustfmt
+    - name: Run clippy
+      uses: actions-rs/clippy-check@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        args: --all --all-features
+    - name: Run rustfmt
```

Comment:
Is it possible to make this run first? It's the fastest check by far (since it doesn't require a build), and if formatting mismatches can be viewed right after it finishes (really quickly), that would be the best.

Reply:
Done, `rustfmt` is the first step of the job. Thanks!